models/template_model.go

```diff
@@ -0,0 +1,117 @@
+package models
+
+import (
+	"database/sql/driver"
+	"time"
+
+	"github.com/percona-platform/saas/pkg/common"
+	"gopkg.in/reform.v1"
+)
+
+//go:generate reform
+
+//reform:ia_templates
+type Template struct {
+	Name        string   `reform:"name,pk"`
+	Version     uint32   `reform:"version"`
+	Summary     string   `reform:"summary"`
+	Tiers       Tiers    `reform:"tiers"`
+	Expr        string   `reform:"expr"`
+	Params      Params   `reform:"params"`
+	For         Duration `reform:"for"`
+	Severity    string   `reform:"severity"`
+	Labels      Map      `reform:"labels"`
```

Comment:
Please use the same approach as Agent:
* https://github.com/percona/pmm-managed/blob/85459abc452472c051f1070374d89dba2073c9be/models/agent_model.go#L74
* https://github.com/percona/pmm-managed/blob/85459abc452472c051f1070374d89dba2073c9be/models/agent_model.go#L140-L148

Reply:
Fixed